actioncable/app/assets/javascripts/action_cable/connection.coffee

```diff
@@ -90,18 +100,18 @@ class ActionCable.Connection
     open: ->
       ActionCable.log("WebSocket onopen event")
       @disconnected = false
```

Comment:
Alright I changed the message handler to check `@isActive` instead. It works the same. I don't quite understand why it worked with `@disconnected` even though as you pointed out there could be a race condition -- whatever underlying threading is going on obviously prioritized the `close` before the `message` is all I can surmise.